### DISCUSSION TITLE
Is WebRTC stream from OvenMediaEngine playable in other WebRTC supported players/browsers?

### DISCUSSION THREAD

**Discussion by deekshithkv** [OvenMediaLabs/OvenMediaEngine#1344] [Help] [answered]
Hi,
Is WebRTC stream produced by OvenMediaEngine playable in other WebRTC protocol supported players otherthan OvenPlayer. Is it a standard WebRTC implementation or custom implementation?

**Comment by bchah** [comment, marked as answer]
It's a custom signalling protocol, described here: https://airensoft.gitbook.io/ovenmediaengine/streaming/webrtc-publishing#signalling

The Airensoft team has already implemented WHIP as a standard for WebRTC inputs, so I bet that more feature requests for WHEP will help them prioritize that standard for output as well 👍

**Reply by deekshithkv** [reply]
Hi,  Is there a general way of implementing WebRTC live streaming or is it generally implemented with custom signaling protocol as in OvenMediaEngine ? And played with custom players? Apart from that is this the same way other media servers also implements WebRTC?

**Reply by bchah** [reply]
The WHEP standard is emerging: https://www.ietf.org/id/draft-murillo-whep-02.html

There are other popular implementations such as Janus and Millicast but these both use concepts outside of the standard.

Here's hoping it all settles down into a real standard soon! 😅

**Reply by deekshithkv** [reply]
hi, Does OvenMediaEngine support H265 codec in their open source or commercial edition?

**Reply by bchah** [reply]
Yes: https://airensoft.gitbook.io/ovenmediaengine/live-source/srt-beta

Requires use of SRT Provider and LLHLS Publisher.

**Reply by deekshithkv** [reply]
For WebRTC streaming is there support for H265 codec? Is there Ovenmedia enterprise edition? if yes, can you share its documentation for features supported. Does it support H265 codec?

**Reply by bchah** [reply]
They sell support and enterprise licenses which allow for further customization outside AGPLv3 license limits: 

https://airensoft.com/support.html

H265 works only as I described for now.

**Reply by getroot** [reply]
No browser supports WebRTC/H.265. This is not a limitation of OvenMediaEngine, but a limitation of the WebRTC standard.

**Reply by deekshithkv** [reply]
we are planning to ingest H.265 IP camera stream in RTSP format to OME and get webRTC stream. Is this possible with OvenMedia open source or enterprise version?

**Reply by getroot** [reply]
@deekshithkv What browser or player are you trying to play WebRTC/H.265 with? As far as I know, no clients support it.
